### ISSUE TITLE
[Feature Request] Allow disabling persistent_notification creation

### ISSUE THREAD

**Issue by edalquist** [FutureTense/keymaster#384]
I have my alarm system arming trigger bolt-checked locking of doors, this creates a persistent_notification every time that I have to manually clear.

I'd like to be able to toggle/disable the creation of persistent_notification entries for operations such as a boltchecked lock request.

**Comment by tykeal** [comment]
I don't believe the "bolt-checked" feature exists anymore with the current version of Keymaster.

**Comment by tykeal** [comment]
After some re-evaluation this does seem to be an actual issue. If the persistent notification is generated because of a `retry lock` event, then it should be cleared if the lock does lock.
